scripts/install-silverstripe.sh

```diff
@@ -1,26 +1,30 @@
 #!/usr/bin/env bash
 
 INSTALL_DIR=/vagrant/www/
-
+INSTALL_DESTINATION=/var/www/html
```

Comment:
Some explanation here about what `INSTALL_DIR` does vs. `INSTALL_DESTINATION` would be helpful.